utils/DeployedContractsHelpers.ts

```diff
@@ -328,45 +359,52 @@ export async function mintWeth(hre: HardhatRuntimeEnvironment, user: SignerWithA
   await weth.connect(user).deposit({ value: amount });
 }
 
-export async function getOnChainBtcEurPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoFiatPrice(hre, constants.EUR_USD_FEED_ADDRESS[networkName], constants.BTC_USD_FEED_ADDRESS[networkName]);
+export async function getOnChainBtcStablePrice(hre: HardhatRuntimeEnvironment, stableSymbol: string) {
+  return getOnChainCryptoFiatPrice(hre, stableSymbol, "BTC");
 }
 
-export async function getOnChainEthEurPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoFiatPrice(hre, constants.EUR_USD_FEED_ADDRESS[networkName], constants.ETH_USD_FEED_ADDRESS[networkName]);
+export async function getOnChainEthStablePrice(hre: HardhatRuntimeEnvironment, stableSymbol: string) {
+  return getOnChainCryptoFiatPrice(hre, stableSymbol, "ETH");
 }
 
 export async function getOnChainWethUsdPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoUSDPrice(hre, constants.ETH_USD_FEED_ADDRESS[networkName]);
+  return getOnChainCryptoUSDPrice(hre, "ETH");
 }
 
 export async function getOnChainWbtcUsdPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoUSDPrice(hre, constants.BTC_USD_FEED_ADDRESS[networkName]);
+  return getOnChainCryptoUSDPrice(hre, "BTC");
 }
 
-export async function getOnChainCryptoUSDPrice(hre: HardhatRuntimeEnvironment, cryptoUsdFeedAddress: string) {
-  const cryptoUsdFeed = (await hre.ethers.getContractAt("AggregatorV3Interface", formatAddress(hre, cryptoUsdFeedAddress))) as AggregatorV3Interface;
+export async function getOnChainCryptoUSDPrice(hre: HardhatRuntimeEnvironment, cryptoSymbol: "ETH" | "BTC") {
+  const networkName = hre.network.name;
+  const cryptoToUsdFeedAddress = _.get(constants, [`${cryptoSymbol}_USD_FEED_ADDRESS`, networkName]);
```

Comment:
because `constants[`${cryptoSymbol}_USD_FEED_ADDRESS`][networkName]
` will throw exception if `constants[`${cryptoSymbol}_USD_FEED_ADDRESS`]` doesn't exist

Reply:
In which case does it not exist?

Reply:
in case someone will misuse it since we use the sent parameter to get the constant value.

1. `lodash` is the most popular package in npm, more than react and express, if you are not familiar with its syntax then now it's a good time to be
2. TS doesn't allowing to do what you are suggesting here
![image](https://user-images.githubusercontent.com/44050297/163357809-a2ee769a-e931-4b69-9517-c31e60ea7944.png)
![image](https://user-images.githubusercontent.com/44050297/163357845-d1b2ed20-0e11-438f-aa59-2b9902b8ef5d.png)